interface/H4LTools.h

```diff
@@ -248,7 +269,7 @@ class H4LTools {
 
 H4LTools::H4LTools(){
   std::string DATAPATH = "";
-  DATAPATH += "KalmanMuonCalibrationsProducer/data/roccor.Run2.v5/RoccoR2017UL.txt";
+  DATAPATH += "KalmanMuonCalibrationsProducer/data/roccor.Run2.v5/RoccoR2018UL.txt";
```

Comment:
don't hard code this here. 

Try to pass the year from here: https://github.com/ram1123/nanoAOD_skim/blob/7b26df21bb996afee7b47d960462e894eff94f9a/post_proc.py

Then use the path based on year.

If (year = 2017) {
DATAPATH += "KalmanMuonCalibrationsProducer/data/roccor.Run2.v5/RoccoR2017UL.txt";
}
else If (year = 2018) {
DATAPATH += "KalmanMuonCalibrationsProducer/data/roccor.Run2.v5/RoccoR2018UL.txt";
}
else  {
 ERROR
}